### PR TITLE
First pull (see description for changes)

### PR DESCRIPTION
**Pull request by nathanschram**
1. Blue Reef Logo
3. Edited Menu bar
2. Inserted sample content

### CODE REVIEW

**Comment by iamgaru** [comment]
At first glance, all good - I am assuming you're reverting those CSS classes? If not, maybe create a new class and reference that.
Apart from that, just ensure that the logos are transparent png's and best resolution for the fit :-)

File: assets/css/styles.css [hunk 1]
```diff
@@ -191,7 +191,7 @@ input[type="button"] {
 .header.navbar-fixed-top .main-nav .nav .dropdown-menu a {
   color: #fff;
 }
-.header h1.logo {
+.logo {
```
Comment (iamgaru) [inline assets/css/styles.css]: A we losing any formatting from removing the formatting for the 'header h1' elements? Perhaps on another page? Just something to check ;-)

If you want a custom class for the logo then you could create a new one - br-logo for example.

**Comment by nathanschram** [comment]
I updated the logo to png, Nick.

Purely out of ignorance, why is reverting that CCS class important? As far as I can see, the .logo rule isn't used anywhere else.

**Comment by iamgaru** [comment]
CSS class? 

Well it’s not as important if it’s not used anywhere else .. but thats the issue isn’t it - ensuring that is the case. 




Whereas if you just create a new class, then you know it’s only being used in that one place. 


—

**Pull request merged**